lib/Sema/CSSyntacticElement.cpp

```diff
@@ -959,6 +959,20 @@ class SyntacticElementConstraintGenerator
     // they would be handled together with pattern because pattern can
     // inform a type of sequence element e.g. `for i: Int8 in 0 ..< 8`
```

Comment:
Looks like this comment needs tweaking slightly to not mention where clauses now

Reply:
Thanks!

Reply:
Done!